server/current-context.js

```diff
@@ -78,7 +78,28 @@ LoopBackContext.createContext = function(scopeName) {
     process.context[scopeName] = ns;
     // Set up LoopBackContext.getCurrentContext()
     LoopBackContext.getCurrentContext = function() {
-      return ns && ns.active ? ns : null;
+      if (ns && ns.active) {
+        /**
+         * **NOTE**
+         * This only re-binds get and set methods, the most used.
```

Comment:
I am worried about breaking applications that expects `getCurrentContext` to return a value that is not bound yet.

How about introducing a new API for getting the bound context? E.g. `loopbackContext.getBoundContext`? Alternatively, we can add an `options` object to `getCurrentContext`, e.g. `loopbackContext.getCurrentContext({ bind: true })`.

Thoughts?